src/python/tests/seq_region/test_dump.py

```diff
@@ -0,0 +1,296 @@
+
+
+@pytest.fixture(name="db_map", scope="module")
+def fixture_db_map() -> dict[str, str]:
+    """Returns the default db_map."""
+    return get_external_db_map(DEFAULT_EXTERNAL_DB_MAP)
+
+
+def test_fetch_coord_systems(seq_test_db: UnitTestDB) -> None:
+    """Tests the `fetch_coord_system` method."""
+    with seq_test_db.dbc.test_session_scope() as session:
+        coords = list(fetch_coord_systems(session))
+        assert len(coords) == 1
+        coord: CoordSystem = coords[0]
+        assert len(list(coord.seq_region)) == 1
+        assert coord.name == "primary_assembly"
+
+
+def test_fetch_seq_regions(seq_test_db: UnitTestDB) -> None:
+    """Tests the `fetch_seq_regions` method."""
+    with seq_test_db.dbc.test_session_scope() as session:
+        coord_system = list(fetch_coord_systems(session))[0]
+        seqrs = list(fetch_seq_regions(session, coord_system))
+        assert len(seqrs) == 1
+        seqr = seqrs[0]
+        assert seqr.name == "seqA"
+
+
+def test_get_synonyms(seq_test_db: UnitTestDB, db_map: dict[str, str]) -> None:
+    """Tests the `get_synonyms` method."""
+    with seq_test_db.dbc.test_session_scope() as session:
+        _add_test_synonym(session, seq_test_db.dbc.dialect, "synA", "db1", 1)
+        coord_system = list(fetch_coord_systems(session))[0]
+        seqr = list(fetch_seq_regions(session, coord_system))[0]
+        syns = get_synonyms(seqr, db_map)
+        assert len(syns) == 1
+        syn = syns[0]
+        assert syn == {"name": "synA", "source": "db1"}
+
+
+@pytest.mark.parametrize(
+    "bands, expected_kar",
+    [
+        param([], [], id="no karyotype"),
+        param([{}], [{"start": 1, "end": 10}], id="no name/stain"),
+        param([{"name": "bandA"}], [{"start": 1, "end": 10, "name": "bandA"}], id="one band"),
+        param([{"stain": "stainA"}], [{"start": 1, "end": 10, "stain": "stainA"}], id="one stain"),
+        param(
+            [{"stain": "TEL"}],
+            [{"start": 1, "end": 10, "stain": "TEL", "structure": "telomere"}],
+            id="telomere",
+        ),
+    ],
+)
+def test_get_karyotype(seq_test_db: UnitTestDB, bands: list, expected_kar: dict) -> None:
+    """Tests the `get_synonyms` method."""
+    with seq_test_db.dbc.test_session_scope() as session:
+        for band in bands:
+            _add_test_karyotype(session, seq_test_db.dbc.dialect, 1, 10, band.get("name"), band.get("stain"))
+        coord_system = list(fetch_coord_systems(session))[0]
+        seqr = list(fetch_seq_regions(session, coord_system))[0]
+        kar = get_karyotype(seqr)
+        assert kar == expected_kar
+
+
+@pytest.mark.parametrize(
+    "attribs, expected_output",
+    [
+        param({}, None, id="no toplevel"),
+        param({"codon_table": 1}, None, id="no toplevel, other attribs"),
+        param({"toplevel": 1}, {}, id="1 toplevel"),
+        param({"toplevel": 1, "codon_table": 2}, {"codon_table": 2}, id="With codon table"),
```

Comment:
```suggestion
        param({"toplevel": 1, "codon_table": 2}, {"codon_table": 2}, id="with codon table"),
```
You are not using capital case in the other IDs.